src/Tasks/Microsoft.NET.Build.Tasks/Microsoft.NET.Build.Tasks.csproj

```diff
@@ -47,6 +47,11 @@
     <PackageReference Include="NuGet.Build.Tasks.Pack" Version="$(NuGetBuildTasksPackVersion)" ExcludeAssets="All" />
   </ItemGroup>
 
+  <!-- System.Reflection.Metadata is in-box for .NET Core 2.0, so we only need to reference it for .NET Framework -->
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
I'm not a fan of using TargetFrameworkIdentifier conditions since they only work in ItemGroups, not property groups and I don't want to have to explain this when folks try to copy/paste/adjust. Also, this is now inconsistent with the item group just below.